pkg/nsxt/connector_manager.go

```diff
@@ -46,8 +58,13 @@ func (processor remoteBasicAuthHeaderProcessor) Process(req *http.Request) error
 	return nil
 }
 
-// NewNsxtConnector creates a new NSXT connector
-func NewNsxtConnector(nsxtConfig *config.NsxtConfig) (client.Connector, error) {
+// NewConnectorManager creates a new NSXT connector
+func NewConnectorManager(nsxtConfig *config.Config) (*ConnectorManager, error) {
+	cm := &ConnectorManager{}
+	if nsxtConfig == nil {
+		return cm, nil
```

Comment:
This shouldn't be an error?

Reply:
please fix in follow-up PR if this should return error

Reply:
I thought `nsxtConfig == nil` is a valid case that user does not add nsxt/routes/lb configurations in ConfigMap. Maybe I should check `nsxtConfig` before calling `NewConnectorManager` in cloud.go.